apps/design-system/config/docs.ts

```diff
@@ -36,9 +41,9 @@ export const docsConfig: DocsConfig = {
           items: [],
         },
         {
-          items: [],
-          href: '/docs/ui-patterns/accessibility',
```

Comment:
Was wrongly ordered. The `href` was also wrong, since this doesn’t live in `ui-patterns`.